src/librustdoc/clean/types.rs

```diff
@@ -303,7 +303,7 @@ impl Item {
 
 #[derive(Clone, Debug)]
 crate enum ItemKind {
-    ExternCrateItem(Symbol, Option<Symbol>),
+    ExternCrateItem(Option<Symbol>),
```

Comment:
While you're at it, do you think it makes sense to give this field a name so it's clear that it's referring to the source crate name, not the name it's imported as?

```suggestion
    ExternCrateItem {
        /// The crate's name, *not* the name it's imported as.
        src: Option<Symbol>,
    },
```